botocore/data/endpoints.json

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
I think this is actually an argument in favor of the rename -- it forces us (and anyone else relying on `_endpoints.json`) to update code to account for the change.